uaclient/actions.py

```diff
@@ -123,7 +126,14 @@ def enable_entitlement_by_name(
         called_name=name,
         access_only=access_only,
     )
-    return entitlement.enable()
+    ent_ret, reason, apt_update = entitlement.enable()
```

Comment:
Maybe we don't want to solve that issue, but if we run the CLI enable command with multiple services, we will still perform more than one `apt update` on this setup